build.gradle

```diff
@@ -1,18 +1,18 @@
 apply plugin: 'java'
 
-sourceCompatibility = 11
-targetCompatibility = 11
+sourceCompatibility = 17
+targetCompatibility = 17
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    implementation 'org.apache.logging.log4j:log4j-core:2.12.1'
-    implementation 'com.vlkan.log4j2:log4j2-logstash-layout:0.19'
+    implementation 'org.apache.logging.log4j:log4j-core:2.17.0'
+    implementation 'com.vlkan.log4j2:log4j2-logstash-layout:1.0.5'
```

Comment:
I'd remove log4j libraries from the skeleton. Is there any reason to keep them?

Reply:
Yup!

I would vote for maintaining it 😬👼

Reasons why:

- Used in the bast majority of Java projects, so highly probable to be needed just after cloning the skeleton
- Useful for Java ecosystem newcomers in order to do not miss logs from libraries you can be actually using which, because of not having a main logger, would not have a visible output for you
- Consistency with the rest of skeletons where yo have an easy way to log things out